tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -0,0 +1,434 @@
+				'tooltip'			=> esc_html__( 'By default, The Events Calendar\'s "Event Cost" field is hidden when Eventbrite Tickets or Event Tickets Plus is active. Check this box to re-enable it.', 'tribe-extension' ),
+				'validation_type'	=> 'boolean',
+			),
+
+			$this->opts_prefix . 'show_tickets_private_events' => array(
+				'type'				=> 'checkbox_bool',
+				'label'				=> esc_html__( 'Display Tickets for Private Events', 'tribe-extension' ),
+				'tooltip'			=> esc_html__( 'By default, Eventbrite Tickets only displays tickets for events marked Public on Eventbrite. Check this box to also display tickets for the Private events that you have imported.', 'tribe-extension' ),
+				'validation_type'	=> 'boolean',
+			),
+
+		);
+
+		$setting_helper->add_fields(
+			$fields,
+			'imports',
+			'tribe_aggregator_disable_header', // if Event Aggregator is enabled, below Meetup, else below CSV
+			true
+		);
+	}
+
+	/**
+	 * Extension initialization and hooks.
+	 */
+	public function init() {
+		add_action( 'admin_init', array( $this, 'add_settings' ) );
+
+		$before = tribe_get_option( $this->opts_prefix . 'content_after' );
+		if ( ! empty( $before ) ) {
+			add_filter( 'tribe_events_eventbrite_before_the_tickets', array( $this, 'before_iframe' ) );
+		}
+
+		$after = tribe_get_option( $this->opts_prefix . 'content_after' );
+		if ( ! empty( $after ) ) {
+			add_filter( 'tribe_events_eventbrite_after_the_tickets', array( $this, 'after_iframe' ) );
+		}
+
+		$show_tickets_private_events = tribe_get_option( $this->opts_prefix . 'show_tickets_private_events' );
+		if ( ! empty( $show_tickets_private_events ) ) {
+			add_filter( 'tribe_events_eventbrite_the_tickets', array( $this, 'render_tickets_iframe_private_eb_events' ) );
+		}
+
+		add_filter( 'tribe_events_eb_iframe_html', array( $this, 'iframe_height' ), 10, 3 );
+
+		if ( 'en_US' !== $this->get_chosen_eb_locale() ) {
+			add_filter( 'tribe-eventbrite-base_api_url', array( $this, 'api_url' ) );
+
+			add_filter( 'tribe_eb_api_sync_event', array( $this, 'eb_currency_for_update_create' ), 10, 5 );
+		}
+
+		// originally from https://theeventscalendar.com/knowledgebase/use-the-event-cost-field-with-tickets-installed/
+		// cannot use https://github.com/moderntribe/products-extensions/blob/master/tribe-ext-show-cost-field/index.php because that Extension requires Event Tickets Plus active
+		$show_admin_cost_field = tribe_get_option( $this->opts_prefix . 'show_admin_cost_field' );
+		if ( ! empty( $show_admin_cost_field ) ) {
+			add_filter( 'tribe_events_admin_show_cost_field', '__return_true', 100 );
+		}
+
+		// originally from https://theeventscalendar.com/knowledgebase/moving-the-eventbrite-tickets-box/
+		$eb_new_ticket_location = tribe_get_option( $this->opts_prefix . 'ticket_single_location' );
+		$eb_ticket_default_location = 'tribe_events_single_event_after_the_meta';
+		if ( ! empty( $eb_new_ticket_location ) && $eb_ticket_default_location !== $eb_new_ticket_location ) {
```

Comment:
Interesting point. There are situations where it's advantageous to leave the action associated with the original. Makes sense to leave it as you did.